recipes/pypolyagamma-3/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pypolyagamma-3" %}
+{% set version = "0.1.10" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Savvysherpa/pypolyagamma/archive/{{ version }}.tar.gz
+  fn: {{ name }}-{{ version }}.tar.gz
+  sha256: 6da114c6398081e8c5e5efca958e7c6e163acc7c9d0ef2c7448cbc35c3382b2d
+
+build:
+  number: 0
+  # We lack gsl on windows
+  skip: True  # [win]
+  skip: True  # [not py35]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - gcc  # [osx]
+    - gsl
+    - numpy
+  run:
+    - python
+    - numpy
+    - scipy
+
+test:
+  commands:
+    - python -c "import pypolyagamma;"
+
+about:
+  home: https://github.com/Savvysherpa/pypolyagamma-3
```

Comment:
The [original](https://github.com/slinderman/pypolyagamma) needed a few changes (in print syntax) to be python3 compatible. I made the changes and submitted a pull request but there was no response. @Savvysherpa has another [package](https://github.com/Savvysherpa/slda) that depends on pypolyagamma that we want to put on conda forge. Thus the motivation to put pypolyagamma on conda forge ourselves. We were wondering what the reaction to this would be. If it goes against the purpose of conda forge or violates some policy we can try to work something else out.